packages/app/src/components/App/index.js

```diff
@@ -12,9 +12,19 @@ import { updateAppActions } from '../../actions';
 import getBrowserLocale from '../../helpers/getBrowserLocale';
 
 const RainbowFirebaseApp = (props) => {
-    const { app, theme, locale, translations, children, reducers, initialize } = props;
+    const {
+        app,
+        theme,
+        locale,
+        translations,
+        children,
+        reducers,
+        initialize,
+        spinnerChild,
+    } = props;
     const firebaseContext = useMemo(() => ({ app }), [app]);
     const [isLoading, setLoading] = useState(false);
+    const [spinnerParams, setSpinnerParams] = useState({});
```

Comment:
why we we need a state for the spinner, by default the spinner has fix params values and if someone passes a custom spinner we should use that one
```
<RainbowFirebaseApp spinner={<MySpinner />}  >
...
```
```

Reply:
we cannot pass a custom spinner that way, because we would break the logic, we only have to pass the parameters of the spinner and its child in case it has any

Here is an example of a custom spinner: 
`export const appWithInitialize = () => {
    const initialize = async () => {
        showAppSpinner({
            size: 'x-small',
            variant: 'base',
        });
        await sleep(2000);
    };

    return (
        <RainbowFirebaseApp app={app} initialize={initialize}>
            <p>App is ready</p>
        </RainbowFirebaseApp>
    );
};`

or you can see the example of the spinner itself

Reply:
`because it would break the logic` ?? can you elaborate more on this

Reply:
This spinner renders at the app level and there is no use case for allowing spinner customization, only showAppSpinner() without params it's good for now.

Reply:
It seems that I misinterpret what is asked, let me check it before